### ISSUE TITLE
page in frame fails to load after clicking Settings/System 

### ISSUE THREAD

**Issue by jweston2112** [home-assistant/frontend#13739]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

When I click Settings/System I just get a Black in the Frame where I expect the next set of menu items

I have checked this on MS Edge and Chrome and Firefox

This is after loading 2022.9.3

### Describe the behavior you expected

To see the System menu items ... Logs, Hardware, Backups... etc. 

### Steps to reproduce the issue

1.
2.
3.
...

After loading newest Version and Clearing the Browser Cache click Settings / System

### What version of Home Assistant Core has the issue?

2022.9.3

### What was the last working version of Home Assistant Core?

2022.9.2

### In which browser are you experiencing the issue with?

Chrome, MS Edge, FireFox

### Which operating system are you using to run this browser?

windows 10 / Kali Linux

### Javascript errors shown in your browser console/inspector

```txt
18 typography.js:181 
        
       Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'language')
    at i (typography.js:181:1)
    at r.value (ha-state-icon.ts:20:1)
    at r.performUpdate (styles.ts:370:1)
    at r.scheduleUpdate (styles.ts:370:1)
    at r._$EC (styles.ts:370:1)

```
```


### Additional information

_No response_

**Comment by frenck** [comment]
duplicate of https://github.com/home-assistant/frontend/issues/13728

fixed Home Assistant in 2022.9.4

**Comment by jweston2112** [comment]
I looked I guess I should have looked harder... 
Sorry, mijn fout...
